testing/tests/headless.test.js

```diff
@@ -29,8 +29,10 @@ describe("Basic viewing of functional pages", () => {
     await expect(page).toMatchElement("h1", {
       text: "<foo>: Une page de test",
     });
-    await expect(page).toSelect('select[name="language"]', "English (US)");
-    await expect(page).toClick("button", { text: "Change language" });
+    await expect(page).toClick("a.view-in-english", {
+      text: "View in English",
+    });
+    await page.waitForNavigation({ waitUntil: "networkidle2" });
```

Comment:
This line was the cause of https://github.com/mdn/yari/issues/3116
Sorry. I should have caught that in review. You should only use a line like this when you depend on lazy-loading to come in. At least that's what I found when I worked on the headless tests for site-search.